src/main/java/Parser.java

```diff
@@ -0,0 +1,71 @@
+import java.util.Scanner;
+
+/**
+ *  makes sense of the user command
+ */
+public class Parser {
+
+    Scanner scan;
+    Ui ui = new Ui();
```

Comment:
Good design choice to put ui as a field of parser, something I should have done for my iP.